wicketd/src/update_tracker.rs

```diff
@@ -1221,10 +1238,38 @@ impl UpdateDriver {
         ipr_start_receiver: IprStartReceiver,
     ) {
         let mut host_registrar = engine.for_component(UpdateComponent::Host);
+
+        // We can't update cosmo sleds at the moment, just skip this step for now
```

Comment:
This comment seems more applicable to `register_deliver_host_phase1_steps()` - we don't skip this step, right?